coalib/core/Core.py

```diff
@@ -258,25 +258,25 @@ def _schedule_bears(self, bears):
                             None, self._execute_task_with_cache,
                             bear, task)
 
-                    tasks.add(future)
+                    futures.add(future)
 
-                self.running_tasks[bear] = tasks
+                self.running_futures[bear] = futures
 
                 # Cleanup bears without tasks after all bears had the chance to
                 # schedule their tasks. Not doing so might stop the run too
```

Comment:
Wouldn't it be better if the comments were modified a little too? Maybe to indicate what `Future` is just like in the commit description.

Reply:
Good point, however this comment is still valid. One task maps to one future, so this comment is still right (and I guess a bit better to understand when using tasks instead of futures here, as primarily bears emit tasks, not futures. The core turns them into futures to process them, but that's an implementation detail). But I'll revise the comments again :+1: